### ISSUE TITLE
Does not recompile `require`d files

### ISSUE THREAD

**Issue by calebeby** [brunch/stylus-brunch#52]
Hi, when I run `brunch watch`, and change a file that is not `index.styl`, it does not update.

Example:

run `brunch watch`

modify `index.styl`

It says `info: compiled index.styl into main.css in 378 ms`, and browsersync injects the new styles. Good :white_check_mark: 

modify `components/idea-card/_index.styl`, which is `require`d by `index.styl`

I get `info: compiled in 78 ms`, but none of the changes are actually built

I have to restart `brunch watch` to get the changes.

**Comment by shvaikalesh** [comment]
Hey @calebeby, thanks for reaching out.

Please make sure you are using the latest version of Brunch and `stylus-brunch`.
The latest version of Brunch (atleast) compiles dependent file even if modified file matches ignored convention. Even if globs imports are used.

**Comment by calebeby** [comment]
OK. I'll check the versions this weekend, but I don't use brunch anymore.

**Comment by shvaikalesh** [comment]
That is sad to hear. If you have any feedback/suggestion regarding Brunch, we would ❤️  to hear.

**Comment by calebeby** [comment]
I was kind of frustrated by the number of brunch plugins for Jade, and I spent a lot of time trying to figure out which one I wanted. I ended up building my own build tool (https://github.com/glugjs/glugjs)

**Comment by calebeby** [comment]
It has support for more templating engines through jstransformers.

However, it is pre-alpha, and lacking documentation.

**Comment by shvaikalesh** [comment]
Yep, we had an issue with static compilers before `v2.8`, but now we have official https://github.com/brunch/jade-brunch.

**Comment by calebeby** [comment]
Come to think of it, I might add a jstransformers input plugin for brunch.

**Comment by calebeby** [comment]
I wanted to compile Jade files to HTML files and so I tried jaded brunch but it didn't work.

Why does Jade brunch say to use jaded brunch for flat files?

**Comment by shvaikalesh** [comment]
Fixed in https://github.com/brunch/jade-brunch/pull/33, thanks!
jstransformers plugin sounds nice 👍

**Comment by calebeby** [comment]
Brunch is `^2.9.1`

Stylus-brunch is `^2.8.0`

**Comment by calebeby** [comment]
I checked, and this behavior still exists.

**Comment by calebeby** [comment]
Yes, the issue persists.

**Comment by calebeby** [comment]
https://gitlab.com/innovation-academy/www/tree/brunch
